sdk/identity/Azure.Identity/src/Credentials/AzurePipelinesServiceConnectionCredentialOptions.cs

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+
+namespace Azure.Identity
+{
+    /// <summary>
+    /// Options used to configure the <see cref="AzurePipelinesServiceConnectionCredential"/>.
+    /// </summary>
+    public class AzurePipelinesServiceConnectionCredentialOptions : TokenCredentialOptions, ISupportsDisableInstanceDiscovery, ISupportsAdditionallyAllowedTenants, ISupportsTokenCachePersistenceOptions
+    {
+        internal CredentialPipeline Pipeline { get; set; }
+
+        internal MsalConfidentialClient MsalClient { get; set; }
+
+        /// <summary>
+        /// The URI of the TFS collection or Azure DevOps organization.
+        /// </summary>
+        public string CollectionUri { get; set; } = Environment.GetEnvironmentVariable("SYSTEM_TEAMFOUNDATIONCOLLECTIONURI");
```

Comment:
Should these environment variables be put in EnvironmentVariables.cs? If not, it should at least check for empty string and turn them to null as we do in EnvironmentVariables.cs.

Reply:
I debated it, but since these are isolated to this specific credential type, I decided to keep them local. 

The default value, if not set, is null and since we aren't driving any logic based on the values being set or not, I don't think normalizing on null adds anything here.